docs/_data/mesheryctlcommands/lifecycle/system/status/command/status.yaml

```diff
@@ -0,0 +1,4 @@
+name: status
+description: Check status of Meshery and Meshery adapters.
```

Comment:
And Meshery Operator and its controllers.

Reply:
@adithyaakrishna I have added this description too according to Lee..